exercises/concept/annalyns-infiltration/.docs/introduction.md

```diff
@@ -4,7 +4,9 @@ Booleans in Go are represented by the predeclared boolean type `bool`, which val
 It's a defined type.
 
 ```go
-var a bool
+var a bool         // boolean variable 'a' implicitly initialized with 'false'
+speeding := true   // boolean variable 'speeding' initialized with 'true'
+hasError := false  // boolean variable 'hasError' initialized with 'false' 
```

Comment:
Just noting a general inconsistency with the variables. Either rename `a` to something semantically meaningful on the lines above or else have these two be `b` and `c` as we would expect.

Reply:
Fixed. Ended up changing the name of `a` to `closed`.